### DISCUSSION TITLE
使用CommunityToolkit.Mvvm的源代码生成器经常会代码报错，但编译能通过

### DISCUSSION THREAD

**Discussion by dtstu** [BYJRK/DotNet-Discussions#69] [MVVM]
使用CommunityToolkit.Mvvm的源代码生成器经常会代码报错，但编译能通过

**Comment by dtstu** [comment]
报错时查看分析器中的生成器，并没有生成代码，但是编译可以通过

**Reply by BYJRK** [reply]
似乎是VS最近几个版本的bug，因为之前并没有遇到过，最近几个月才开始出现的。应该算是正常现象

**Reply by dtstu** [reply]
> 似乎是VS最近几个版本的bug，因为之前并没有遇到过，最近几个月才开始出现的。应该算是正常现象

试了很多办法都不行，最后退回8.0版本试了下，起飞

**Comment by CrazyZhang666** [comment]
删除obj文件夹，然后重新编译即可
CommunityToolkit 更新到最新版本
